test/e2e_node/pod_resize_test.go

```diff
@@ -18,6 +18,7 @@ package e2enode
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
I refactored pod resize e2e tests in https://github.com/kubernetes/kubernetes/pull/128143. Is it possible to update the test file?